ultra/ultra/scenarios/pool/2lane_c/100kmh/map.net.xml

```diff
@@ -1,226 +1,204 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How similar are these new map files to the old ones? Is it just the extra road segment that is missing? Are all the other edges the same size and at the same coordinates?